boostlook.css

```diff
@@ -137,3 +137,70 @@ padding: 0.25em 0.55em;
 :root {
 --color-pasteboard: rgb(209, 213, 219);
 }
+
+body.article {
+  min-height: 100vh;
+  max-width: 80rem;
+  margin-left: auto;
+  margin-right: auto;
+  padding: 1rem;
+  position: relative;
+  background-color: var(--color-pasteboard);
+}
+
+.boostlook #toc > ul.sectlevel1 {
+  padding-left: 0;
+}
+
+.boostlook #toc ul {
+  margin: 0;
+  padding: 0 0 0 0.5rem;
+  line-height: 1.5;
+  font-size: 0.85rem;
+  list-style: none;
+}
+
+.boostlook #toc .sectlevel1 > li > a {
+  font-weight: bolder;
+}
+
+.boostlook #toc #toctitle {
+  font-size: 1.5rem;
+}
+
+.boostlook #toc ul li a {
+  font-size: 1rem;
+}
+
+.boostlook,
+.boostlook #toc.toc2 {
+  background-color: white;
+  border-radius: 0.5rem;
+  padding: 1rem;
+}
+
+.boostlook #toc.toc2 {
+  padding-left: 0;
+}
+
+.boostlook #content {
+  overflow-x: auto;
+}
+
+@media screen and (min-width: 768px) {
```

Comment:
This should be using container query

Reply:
Tried making it work with the @container query, I'm not too familiar with it btw. I'm going to revisit how we can leverage that query in future PRs.